config/overlays/odh/kustomization.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../crd
+  - ../../prometheus
+  - ./default
+  - ./webhook
+  - ./scripts
+  - ./quickstart.yaml
+  - ./rbac
+
+namespace: opendatahub
+configMapGenerator:
+  - envs:
+      - params.env
+    name: mesh-parameters
+generatorOptions:
+  disableNameSuffixHash: true
+
+transformers:
+  - ./default/metadataLabelTransformer.yaml
+
+vars:
+  - fieldref:
+      fieldPath: metadata.namespace
+    name: mesh-namespace
+    objref:
+      apiVersion: v1
+      kind: ConfigMap
+      name: mesh-parameters
+  - fieldref:
+      fieldPath: data.odh-modelmesh
+    name: odh-modelmesh
+    objref:
+      apiVersion: v1
+      kind: ConfigMap
+      name: mesh-parameters
+  - fieldref:
+      fieldPath: data.odh-mm-rest-proxy
+    name: odh-mm-rest-proxy
+    objref:
+      apiVersion: v1
+      kind: ConfigMap
+      name: mesh-parameters
+  - fieldref:
+      fieldPath: data.odh-modelmesh-runtime-adapter
+    name: odh-modelmesh-runtime-adapter
+    objref:
+      apiVersion: v1
+      kind: ConfigMap
+      name: mesh-parameters
+  - fieldref:
+      fieldPath: data.odh-modelmesh-controller
+    name: odh-modelmesh-controller
+    objref:
+      apiVersion: v1
+      kind: ConfigMap
+      name: mesh-parameters
+  - fieldref:
+      fieldPath: data.odh-openvino
+    name: odh-openvino
+    objref:
+      apiVersion: v1
+      kind: ConfigMap
+      name: mesh-parameters
```

Comment:
Unused variable. Please, remove.

Reply:
I started a discussion about this internally, this variable was also unused previously but currently it is the only place in ODH where we are specifying the openvino variable. Removing this can be a separate PR

Reply:
If there are no hard reasons to keep it, let's remove it here. I prefer to have it cleaned up.

The unused variable generated a log entry in the operator pod. The cleaner the logs, the better.

Reply:
@VedantMahabaleshwarkar I agree with @israel-hdez 
we can remove odh-openvino related (2 files)
- config/overlays/odh/kustomization.yaml#56~#62
- config/overlays/odh/params.env#4